pensieve/tests/integration/test_analysis_integration.py

```diff
@@ -112,9 +113,7 @@ def build_query_test_project(instance, *args, **kwargs):
             select_expr=agg_sum("active_hours_sum"),
         )
 
-        config.metrics.daily = []
-        config.metrics.weekly = [test_active_hours]
-        config.metrics.overall = []
+        config.metrics = {AnalysisPeriod.WEEK: test_active_hours}
```

Comment:
Does this need to be
```suggestion
        config.metrics = {AnalysisPeriod.WEEK: [test_active_hours]}
```